### PR TITLE
Seens refactor

### PR DESCRIPTION
**Pull request by justinamaple**

### CODE REVIEW

File: app/controllers/letters_controller.rb [hunk 1]
```diff
@@ -40,8 +46,4 @@ def update
   def set_letter
     @letter = Letter.find(params[:id])
   end
-
-  def letter_params
```
Comment (kara-o) [inline app/controllers/letters_controller.rb]: not needed?

Comment (kara-o) [reply app/controllers/letters_controller.rb]: i think it is still called in update

File: app/controllers/letters_controller.rb [hunk 2]
```diff
@@ -1,8 +1,14 @@
+# frozen_string_literal: true
+
 class LettersController < ApplicationController
-  before_action :set_letter, only: [:show, :update]
+  before_action :set_letter, only: %i[show update]
```
Comment (kara-o) [inline app/controllers/letters_controller.rb]: had to look %i up :)

File: app/controllers/responses_controller.rb [hunk 3]
```diff
@@ -1,5 +1,13 @@
+# frozen_string_literal: true
+
 class ResponsesController < ApplicationController
+  before_action :set_letter, only: %i[index]
   before_action :set_response, only: %i[show]
+
+  def index
```
Comment (kara-o) [inline app/controllers/responses_controller.rb]: great!!

File: app/controllers/responses_controller.rb [hunk 4]
```diff
@@ -1,5 +1,13 @@
...
+    render json: @letter.responses
```
Comment (kara-o) [inline app/controllers/responses_controller.rb]: great!!

File: app/controllers/letters_controller.rb (hunk 1 above)
Comment (justinamaple) [reply app/controllers/letters_controller.rb]: Oops, good catch!

**Pull request merged**